test/unit/client-daemonset.bats

```diff
@@ -569,9 +569,31 @@ load _helpers
       --set 'connectInject.enabled=true' \
       . | tee /dev/stderr |
       yq -r '.spec.template.metadata.annotations."consul.hashicorp.com/config-checksum"' | tee /dev/stderr)
-  [ "${actual}" = 8496f6bcdec460eac8a5c890e7899f5757111e13e54808af533aaf205ef18bd0 ]
+  [ "${actual}" = e9ab414837a3c551f398da62e80fad73a2f1e3c0d0ca82e1067bb4ff68ba6aa9 ]
 }
 
+@test "client/DaemonSet: adds extra-from-vals by default" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/client-daemonset.yaml  \
+      --set 'global.acls.manageSystemACLs=true' \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.containers[] | select(.name=="consul") | .command | join(" ") | contains("{}")' | tee /dev/stderr)
```

Comment:
I think we also need to test that the `-config-file` with the right file is passed in